### PR TITLE
Run codespell on codebase

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by edsantiago** [comment]
Oh joy. @cevich the graphql thing again.
>ERROR: Failed: /var/tmp/go/src/github.com/containers/podman/.github/actions/check_cirrus_cron/cron_failures.sh with output '::group::Posting GraphQL Query and checking result query: { "query": "query { ownerRepository(platform: \"LINUX\", owner: \"containers\", name: \"podman\") { cronSettings { name lastInvocationBuild { id status } } }}" } ::error file=/var/tmp/go/src/github.com/containers/podman/.github/actions/check_cirrus_cron/cron_failures.sh,line=49::Query result did not pass filter '.data.ownerRepository.cronSettings': '{"data":{"ownerRepository":null}}' ::endgroup:: '  (/var/tmp/go/src/github.com/containers/podman/.github/actions/check_cirrus_cron/test.sh:54 in main())

**Comment by edsantiago** [comment]
Oops, never mind. @rhatdan please rebase.

**Review by giuseppe — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -275,7 +275,7 @@ test/version/version: version/version.go
 
 .PHONY: codespell
 codespell:
-	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L te,clos,ans,pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether,specif -w
+	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L hastable,te,clos,ans,pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether,specif -w
```
Comment (edsantiago) [inline Makefile:278]: Why this addition? I can't find the string "hastable" anywhere in `main`; I can't even find it in `git log -S`

Comment (rhatdan) [reply Makefile:278]: codespell wants to change HasTable -> HashTable.
./cmd/podman/containers/ps.go:		noHeading = noHeading || !report.HasTable(listOpts.Format)
./cmd/podman/images/list.go:		if cmd.Flags().Changed("format") && !report.HasTable(listFlag.format) {

**Pull request merged**